test/core/Graphics.js

```diff
@@ -128,4 +128,33 @@ describe('PIXI.Graphics', () =>
             expect(graphics.containsPoint(point)).to.be.false;
         });
     });
+
+    describe('complex drawing #1: draw triangle, rounder rect and an arc (issue #3433)', () =>
```

Comment:
Ok, just added requested changes.

But as i described in the issue, it is possible that something else is broken within Graphics module. I just found a case, when things blow up on `.arc(..)` call, but i also described another case when things do not blow up, but the outcome is not correct (as one can see on the attached screenshot).

So, may be it is a good idea to add more tests for the module. I could do it, probably as another PR, but i do not know how the API is meant to work internally.